cmd/curator/curator.go

```diff
@@ -37,8 +37,6 @@ func buildApp() *cli.App {
 		operations.HelloWorld(),
 		operations.Version(),
 		operations.S3(),
-		operations.Repo(),
```

Comment:
Don't we still need the `repo submit` command?

Reply:
Or that is in barque now?

Reply:
Ah yeah, you're right. I forgot that the barque submission command was a subcommand of the repo command.